src/plugins/data/public/ui/query_editor/editors/shared.tsx

```diff
@@ -150,7 +153,7 @@ export const SingleLineInput: React.FC<SingleLineInputProps> = ({
           triggerSuggestOnFocus={true}
         />
         {editorIsFocused && (
-          <div className="queryEditor__footer">
+          <div className="queryEditor__footer" data-test-subj="queryEditorFooter">
```

Comment:
If I remember correctly, I added this for the single line editor's footer. Could be a little more descriptive for this to avoid confusion with the other editors.